examples/on_gke_end_to_end/main.tf

```diff
@@ -0,0 +1,172 @@
+ */
+
+//*****************************************
+//  Setup Google providers
+//*****************************************
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+provider "google-beta" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+//*****************************************
+//  Setup the Kubernetes Provider
+//*****************************************
+
+data "google_client_config" "default" {}
+
+provider "kubernetes" {
+  alias                  = "forseti"
+  load_config_file       = false
+  host                   = "https://${module.gke.endpoint}"
+  token                  = "${data.google_client_config.default.access_token}"
+  cluster_ca_certificate = "${base64decode(module.gke.ca_certificate)}"
+}
+
+//*****************************************
+//  Setup Helm Provider
+//*****************************************
+
+provider "helm" {
+  alias           = "forseti"
+  service_account = "${var.k8s_tiller_sa_name}"
+  namespace       = "${var.k8s_forseti_namespace}-${module.forseti.suffix}"
+  kubernetes {
+    load_config_file       = false
+    host                   = "https://${module.gke.endpoint}"
+    token                  = "${data.google_client_config.default.access_token}"
+    cluster_ca_certificate = "${base64decode(module.gke.ca_certificate)}"
+  }
+  debug                           = true
+  automount_service_account_token = true
+  install_tiller                  = true
+}
+
+module "vpc" {
+  source                  = "terraform-google-modules/network/google"
+  version                 = "1.1.0"
+  project_id              = "${var.project_id}"
+  network_name            = "${var.network_name}"
+  routing_mode            = "GLOBAL"
+  description             = "${var.network_description}"
+  auto_create_subnetworks = "${var.auto_create_subnetworks}"
+
+  subnets = [{
+    subnet_name   = "${var.sub_network_name}"
+    subnet_ip     = "${var.gke_node_ip_range}"
+    subnet_region = "${var.region}"
+  }, ]
+
+  secondary_ranges = {
+    "${var.sub_network_name}" = [
+      {
+        range_name    = "gke-pod-ip-range"
+        ip_cidr_range = "${var.gke_pod_ip_range}"
+      },
+      {
+        range_name    = "gke-service-ip-range"
+        ip_cidr_range = "${var.gke_service_ip_range}"
+      },
+    ]
+  }
+}
+
+module "forseti" {
+  source                  = "../../"
+  gsuite_admin_email      = "${var.gsuite_admin_email}"
+  domain                  = "${var.domain}"
+  project_id              = "${var.project_id}"
+  org_id                  = "${var.org_id}"
+  network                 = "${module.vpc.network_name}"
+  subnetwork              = "${var.sub_network_name}"
+  client_private          = "${var.client_private}"
+  server_private          = "${var.server_private}"
+  storage_bucket_location = "${var.region}"
+  server_region           = "${var.region}"
+  client_region           = "${var.region}"
+  cloudsql_region         = "${var.region}"
+}
+
+module "gke" {
```

Comment:
This module should be pinned to the version 4 releases to ensure Terraform 0.12 compatibility.